content/Resources/GPU_Resources/index.md

```diff
@@ -0,0 +1,13 @@
+# CMS GPU Resources
+
+# Facilities
+
+1. Collaboration-wide GPU resources
```

Comment:
I don't think you need to specify that these are collaboration wide resources. This is implied as there are no restricted resources listed right now. I would simply number the four sub-items.

Reply:
Thanks for your comment, I use a numbered list instead.